Apakah [1, 3] adalah eigen vector dari [[1 −1], [6 4]]? Jika Iya, berapakah eigen value-nya?

In [17]:
import numpy as np
m = [[1, -1], [6, 4]]
mat_m = np.array(m) #Ubah ke array

valE, vecE = np.linalg.eig(mat_m) #Mengambil nilai eigen value dan eigen vector dari matriks
print(valE) #Eigen value
print(vecE) #Eigen vector
print('\n')

print("Eugen Value 1 :", valE[0].real)
print("Eugen Value 2 :", valE[1].real)
print("Eugen Vector 1 :", vecE[:,0].real.round(2))
print("Eugen Vector 2 :", vecE[:,0].real.round(2))

[2.5+1.93649167j 2.5-1.93649167j]
[[-0.23145502+0.29880715j -0.23145502-0.29880715j]
 [ 0.9258201 +0.j          0.9258201 -0.j        ]]


Eugen Value 1 : 2.5
Eugen Value 2 : 2.5
Eugen Vector 1 : [-0.23  0.93]
Eugen Vector 2 : [-0.23  0.93]


Matriks [[1, -1], [6, 4]] mempunyai nilai value eigen, yaitu 2,5. Untuk vector eigen-nya adalah [-0.23  0.93]. Sehingga dapat dikatakan vector [1, 3] bukan eigen vector dari matriks [[1 −1], [6 4]]